packages/eslint-plugin-next/lib/rules/missing-preload.js

```diff
@@ -19,7 +19,7 @@ module.exports = {
             context.report({
               node: linkNode,
               message:
-                'Stylesheet does not have an associated preload tag. This could potentially impact First paint.',
+                'Stylesheet does not have an associated preload tag. This could potentially impact First paint. See: https://web.dev/preload-critical-assets/.',
```

Comment:
It might be good to add docs in the top-level `errors` folder https://github.com/vercel/next.js/tree/canary/errors for these and link to them via https://nextjs.org/docs/messages to allow to us to provide the direct fix needed to resolve the message and then provide additional resources as links in the document.

Reply:
Ooh great point, didn't even know this folder existed 😅 

Done!